PSReadLine/Options.cs

```diff
@@ -271,24 +271,41 @@ public static void RemoveKeyHandler(string[] key)
         /// </summary>
         public static IEnumerable<PowerShell.KeyHandler> GetKeyHandlers()
         {
-            return GetKeyHandlers(includeBound: true, includeUnbound: false);
+            return GetKeyHandlers(includeBound: true, includeUnbound: false, Chord: null);
         }
 
         /// <summary>
-        /// Helper function for the Get-PSReadLineKeyHandler cmdlet.
+        /// Return bound/unbound key handlers without filtering.
         /// </summary>
         /// <returns></returns>
         public static IEnumerable<PowerShell.KeyHandler> GetKeyHandlers(bool includeBound, bool includeUnbound)
+        {
+            return GetKeyHandlers(includeBound: includeBound, includeUnbound: includeUnbound, Chord: null);
+        }
+
+        /// <summary>
+        /// Helper function for the Get-PSReadLineKeyHandler cmdlet.
+        /// </summary>
+        /// <returns></returns>
+        public static IEnumerable<PowerShell.KeyHandler> GetKeyHandlers(bool includeBound, bool includeUnbound, string[] Chord)
         {
             var boundFunctions = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
 
+            var searchChords = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
+
+            if (Chord != null) { foreach (var Key in Chord) { searchChords.Add(Key); }; }
```

Comment:
This means you don't different keys like "Ctrl+k" from "Ctrl+K", but they are different key bindings.
For "Ctrl+k", it's triggered by pressing <kbd>Ctrl</kbd> and <kbd>k</kbd>
For "Ctrl+K", it's triggered by pressing <kbd>Ctrl</kbd>, <kbd>Shift</kbd>, and <kbd>k</kbd>

To accurately find the key, you will need to do the similar to [this code](https://github.com/PowerShell/PSReadLine/blob/03a0d3fe2b715eee66dcdc1304cac9e03456b434/PSReadLine/Options.cs#L178-L179), to get the `PSKeyInfo` that the specified string chord represents.

Reply:
Implemented